azure-pipelines-file-uploader.yml

```diff
@@ -0,0 +1,79 @@
+variables:
+  dockerVersion: '19.03.5'
+  dotnetVersion: '3.1.101'
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+
+- task: DockerInstaller@0
+  inputs:
+    dockerVersion: $(dockerVersion)
+  displayName: 'Docker: Install - $(dockerVersion)'
+
+- task: UseDotNet@2
+  displayName: 'Use DotNet Core $(dotnetVersion)'
+  inputs:
+    packageType: 'sdk'
+    version: $(dotnetVersion)
+    includePreviewVersions: false
+
+- script: docker-compose -f "docker-compose.integration.yml" up --build -d
+  displayName: 'Docker-compose up'
+
+- task: DotNetCoreCLI@2
+  displayName: 'Run DAPI Integration tests'
+  inputs:
+    command: test
+    projects: '**/*Integration.Tests/*.csproj'
+    arguments: '-v n'
+
+
+- script: docker-compose -f "docker-compose.integration.yml" down -v
+  displayName: 'Docker-compose down'
+  condition: always()
```

Comment:
I'm not quite sure that this is necessary / useful.

DAPIs integration tests do not use the file-uploader in any shape or form, so running them isn't going to test anything.

I'd suggest removing this part. What do you think?

Reply:
oh, i thought the integration tests used those documents. if not then I'll pull them out

Reply:
The integration tests are independent and set up using [this](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueDocumentService/blob/3439c98e676529045e8f252f8ac3c5c51e6cd6e1/test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Steps/AzureBlobSteps.cs#L19)
where they can insert data from the 'SampleData' directory into the  blob store container.